tests/test_compute_funs.py

```diff
@@ -1298,6 +1299,159 @@ def test_compute_everything():
     assert not error
 
 
+@pytest.mark.unit
+def test_compute_everything_xyz():
```

Comment:
we could combine this with the previous one and use `pytest.mark.parametrize`

Would also be good to explicitly say `basis="rpz"` in the previous one.

Reply:
Yeah I also thought of that. I will combine them